AndroidProject/app/.gitignore

```diff
@@ -0,0 +1,85 @@
+# Built application files
```

Comment:
This file should be placed in a root folder of your repository

Reply:
done